### PR TITLE
I3S-Debug - Show normals in percentage way

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ import {
 } from './normals-utils';
 
 const TRANSITION_DURAITON = 4000;
-const DEFAULT_NORMALS_GAP = 30; // Gap for normals visualisation to avoid mess on the screen.
+const DEFAULT_NORMALS_PERCENTAGE = 30; // Gap for normals visualisation to avoid mess on the screen.
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:52]: Edit the comment?

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 2]
```diff
@@ -163,7 +163,7 @@ export default class App extends PureComponent {
       selectedMapStyle: INITIAL_MAP_STYLE,
       debugOptions: INITIAL_DEBUG_OPTIONS_STATE,
       normalsDebugData: [],
-      normalsGap: DEFAULT_NORMALS_GAP,
+      normalsPercentage: DEFAULT_NORMALS_PERCENTAGE,
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:166]: Comment? Percentage of what? Total number of normals?

**Review by ibgreen — APPROVED** [review]
Ideally we should create a normals layer that renders normals on the GPU so we don't need to do all the processing on CPU, in which case I think we could drop the percentage unless that is just a visual clutter thing. 

Though short normals (like 1m) should improve considerably. I don't think normals need to be visible when zoomed out.

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js]: Renamed

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js]: Fixed. It is a percentage of triangles to show normals for.

**Pull request merged**